docs/sources/datasources/azure-monitor/_index.md

```diff
@@ -211,6 +211,10 @@ The public cloud name is `AzureCloud`, the Chinese national cloud name is `Azure
 
 ### Configure Managed Identity
 
+{{< admonition type="note" >}}
+Managed Identity is available only in [Azure Managed Grafana](https://azure.microsoft.com/en-us/products/managed-grafana) or Grafana Enterprise when deployed in Azure. It is not available in Grafana Cloud.
```

Comment:
Not limited to Grafana Enterprise. Also available in Grafana OSS running on Azure compute.